### PR TITLE
[#175012163] Hide tags on dialogues and resolution pages 

### PR DESCRIPTION
**Pull request by Vadim-Kurnatovskiy**
[PT](https://www.pivotaltracker.com/story/show/175012163)

### CODE REVIEW

File: plugin.rb [hunk 1]
```diff
@@ -89,7 +89,8 @@
   end
 
   on(:topic_created) do |topic, opts, _user|
-    topic.tags.find_or_create_by(name: opts[:is_resolution] ? "resolution" : "dialogue")
+    tag = Tag.find_or_create_by!(name: opts[:is_resolution] ? "resolution" : "dialogue")
```
Comment (DmitryBarskov) [inline plugin.rb]: I don't think wee need "resolution" tag anymore.

File: assets/javascripts/discourse/templates/composer.hbs [hunk 2]
```diff
@@ -0,0 +1,225 @@
...
+                      tabindex="3"
+                      onChange=(action (mut model.categoryId))
+                      isDisabled=disableCategoryChooser
+                      options=(hash
+                        scopedCategoryId=scopedCategoryId
+                      )
+                    }}
+                    {{popup-input-tip validation=categoryValidation}}
+                  </div>
+                {{/if}}
```
Comment (DmitryBarskov) [inline assets/javascripts/discourse/templates/composer.hbs:86]:
```suggestion
                  </div>
                  {{!-- tag chooser deleted --}}
                {{/if}}
```

File: assets/javascripts/discourse/templates/composer.hbs [hunk 3]
```diff
@@ -0,0 +1,225 @@
...
+                 typed=(action "typed")
+                 cancelled=(action "cancelled")
+                 save=(action "save")}}
+  <div class="grippie"></div>
+  {{#if visible}}
+    {{composer-messages composer=model
+                        messageCount=messageCount
+                        addLinkLookup=(action "addLinkLookup")}}
+    {{#if model.viewOpenOrFullscreen}}
+      <div role="form" aria-label={{I18n saveLabel}} class="reply-area">
```
Comment (DmitryBarskov) [inline assets/javascripts/discourse/templates/composer.hbs]:
```suggestion
      <div role="form" aria-label={{I18n saveLabel}} class="reply-area">{{!-- with-tags class deleted --}}
```

**Review by DmitryBarskov — APPROVED** [review]
few optional notes

File: plugin.rb (hunk 1 above)
Comment (Vadim-Kurnatovskiy) [reply plugin.rb]: i think we can leave it

**Pull request merged**